File: PROYECTO/PSport-APP/PSport/Model/DTOs/ReservaRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PSport.Model.DTOs
{
    public class ReservaRequest
    {

        public Guid id { get; set; }
        public string fecha { get; set; }
        public string horaInicio { get; set; }
        public string horaFin { get; set; }
        public string pistaID { get; set; }
        public string usuarioID { get; set; }
    }
}

File: PROYECTO/PP-Sport-API-/PPSportAPI/src/main/kotlin/es/sergigavi/PPSportAPI/MODEL/REQUEST/ReservaRequest.kt
package es.sergigavi.PPSportAPI.MODEL.REQUEST
import java.time.LocalDate
import java.time.LocalDateTime
import java.time.LocalTime
import java.util.*

data class ReservaRequest(
    val fecha: LocalDate,
    val horaInicio: LocalTime,
    val horaFin: LocalTime,
    val pistaID: UUID,
    val usuarioID: UUID
)
